src/Encryption/Security/JWT/Token/Parser.php

```diff
@@ -100,7 +100,7 @@ private function decodeHeaders(string $headers): Item
 
         if (true !== isset($decoded[Enum::TYPE])) {
             throw new InvalidArgumentException(
-                'Invalid Header (missing "typ" element)'
+                "Invalid Header (missing 'typ' element)"
```

Comment:
```suggestion
                "Invalid Header (missing 'type' element)"
```

`type`?

Reply:
No it is called `typ` - short for type. That is according to the RFC